File: QTD2.Domain/Entities/Core/Version_Task_EnablingObjective_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_Task_EnablingObjective_Link : Common.Entity
    {
        public int Version_EnablingObjectiveId { get; set; }

        public int Version_TaskId { get; set; }

        public string VersionNumber { get; set; }

        public virtual Version_EnablingObjective Version_EnablingObjective { get; set; }

        public virtual Version_Task Version_Task { get; set; }

        public Version_Task_EnablingObjective_Link()
        {
        }

        public Version_Task_EnablingObjective_Link(int version_EnablingObjectiveId, int version_TaskId, string versionNumber)
        {
            Version_EnablingObjectiveId = version_EnablingObjectiveId;
            Version_TaskId = version_TaskId;
            VersionNumber = versionNumber;
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/Version_Task_EnablingObjective_Link/Version_Task_EnablingObjective_Link.ts
import { Entity } from '../Entity';
import { Version_EnablingObjective } from '../Version_EnablingObjective/Version_EnablingObjective';
import { Version_Task } from '../Version_Task/Version_Task';

export class Version_Task_EnablingObjective_Link extends Entity {
  version_EnablingObjectiveId!: any;
  version_TaskId!: any;
  version_EnablingObjective!: Version_EnablingObjective;
  version_Task!: Version_Task;
}
